### PR TITLE
Issue 295: LocalBookKeeper fails to start up

### PR DESCRIPTION
**Pull request by zhaijack**
Descriptions of the changes in this PR:

remote the zk port and zk servers from the classes and use the values passed in from the method.

### CODE REVIEW

**Comment by eolivelli** [comment]
I have restarted the job 
https://builds.apache.org/job/bookkeeper-precommit-pullrequest/174/

**Comment by eolivelli** [comment]
Jenkins is broken....I will try on my laptop

**Comment by ivankelly** [comment]
I came across this the other day. It failed to start because there was setLoopbackAllowed needed to be set in the configuration.

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/util/LocalBookKeeper.java [hunk 1]
```diff
@@ -299,6 +297,9 @@ static void startLocalBookiesInternal(ServerConfiguration conf,
                 }
                 throw ie;
             }
+        } catch (Exception e) {
```
Comment (merlimat) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/util/LocalBookKeeper.java]: If we catch the exception here, it means we are not exiting the process. I would say to not catch here.

Comment (sijie) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/util/LocalBookKeeper.java]: +1 on this. if we want logging, we should re-throw it again.

**Comment by eolivelli** [comment]
I Am sorry. I have already closed the pr. I will create a new one and do the proposed fix from @merlimat

**Pull request closed without merging**